Sources/PointFree/Admin/Index.swift

```diff
@@ -91,6 +91,7 @@ private func sendEmail(forNewEpisode episode: Episode, toUsers users: [Database.
       sendEmail(
         to: [user.email],
         subject: "New Point-Free Episode: \(episode.title)",
+        unsubscribeData: (user.id, .newEpisode),
```

Comment:
if you wanna have the `List-Unsubscribe` header show you, you gotta include the user id and newsletter.

I think soon we need to do a small refactor on email to make it a bit more testable and nice. right now there's a bit of separation between the view that represents the email, the full email data package (subject, recipients, etc...), and the actual sending of the email. if we unify those things it will be much nicer.